docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -102,6 +102,47 @@ Sub-sections related to the Administration Server, specific clusters, or specifi
 
 The elements `serverStartPolicy`, `serverStartState`, `serverPod` and `serverService` are repeated under `adminServer` and under each entry of `clusters` or `managedServers`.  The values directly under `spec` set the defaults for the entire domain.  The values under a specific entry under `clusters` set the defaults for cluster members of that cluster.  The values under `adminServer` or an entry under `managedServers` set the values for that specific server.  Values from the domain scope and values from the cluster (for cluster members) are merged with or overridden by the setting for the specific server depending on the element.  See the [startup and shutdown]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md" >}}) documentation for details about `serverStartPolicy` combination.
 
+### JVM Memory and Java Option Environment Variables
```

Comment:
JVM Memory and Java Option Environment Variables -> JVM memory and Java option environment variables (we generally use sentence capitalization instead of title capitalization)